src/js_printer.zig

```diff
@@ -3982,338 +3987,338 @@ fn NewPrinter(
-                    p.printIndent();
-                    p.printSpaceBeforeIdentifier();
-                    p.addSourceMapping(stmt.loc);
-                    p.print("while");
-                    p.printSpace();
-                    p.print("(");
-                    p.printExpr(s.test_, .lowest, ExprFlag.None());
-                    p.print(")");
-                    p.printBody(s.body);
+                    try p.printIndent();
+                    try p.printSpaceBeforeIdentifier();
+                    try p.addSourceMapping(stmt.loc);
+                    try p.print("while");
+                    try p.printSpace();
+                    try p.print("(");
+                    try p.printExpr(s.test_, .lowest, ExprFlag.None());
+                    try p.print(")");
+                    try p.printBody(s.body);
                 },
                 .s_with => |s| {
-                    p.printIndent();
-                    p.printSpaceBeforeIdentifier();
-                    p.addSourceMapping(stmt.loc);
-                    p.print("with");
-                    p.printSpace();
-                    p.print("(");
-                    p.printExpr(s.value, .lowest, ExprFlag.None());
-                    p.print(")");
-                    p.printBody(s.body);
+                    try p.printIndent();
+                    try p.printSpaceBeforeIdentifier();
+                    try p.addSourceMapping(stmt.loc);
+                    try p.print("with");
+                    try p.printSpace();
+                    try p.print("(");
+                    try p.printExpr(s.value, .lowest, ExprFlag.None());
+                    try p.print(")");
+                    try p.printBody(s.body);
                 },
                 .s_label => |s| {
                     if (!p.options.minify_whitespace and p.options.indent.count > 0) {
-                        p.printIndent();
+                        try p.printIndent();
                     }
-                    p.printSpaceBeforeIdentifier();
-                    p.addSourceMapping(stmt.loc);
-                    p.printSymbol(s.name.ref orelse Output.panic("Internal error: expected label to have a name {any}", .{s}));
-                    p.print(":");
-                    p.printBody(s.stmt);
+                    try p.printSpaceBeforeIdentifier();
+                    try p.addSourceMapping(stmt.loc);
+                    try p.printSymbol(s.name.ref orelse Output.panic("Internal error: expected label to have a name {any}", .{s}));
+                    try p.print(":");
+                    try p.printBody(s.stmt);
                 },
                 .s_try => |s| {
-                    p.printIndent();
-                    p.printSpaceBeforeIdentifier();
-                    p.addSourceMapping(stmt.loc);
-                    p.print("try");
-                    p.printSpace();
-                    p.printBlock(s.body_loc, s.body, null);
+                    try p.printIndent();
```

Comment:
this is a _lot_ if error-handling.

can we run a quick benchmark and see if there is any performance hit to printing?